packages/tm-grammars/index.d.ts

```diff
@@ -18,6 +24,7 @@ export interface GrammarInfo {
   aliases?: string[]
   licenseUrl?: string
   license?: string
+  funding?: FundingLink[]
```

Comment:
These shouldn't ever be undefined, except the custom `source` function for `ts-tags` is causing it to be undefined sometimes so I just adjusted the type definition since I wasn't sure how to fix that properly

Reply:
I think it's better to be optional. If it's an empty array, we better remove the field to save some space of the generated file